doc-surrealdb_versioned_docs/version-1.x/reference-guide/vector-search.mdx

```diff
@@ -0,0 +1,219 @@
+---
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Vector Search
+
+SurrealDB supports [Full-Text Search](/docs/surrealdb/reference-guide/full-text-search) and Vector Search. Full-Text search(FTS) involves indexing documents using the [FTS index](/docs/surrealdb/surrealql/statements/define/indexes#full-text-search-index) and breaking down the content of the document into smaller tokens with the help of [analysers](/docs/surrealdb/surrealql/statements/define/analyzer) and [tokenizers](/docs/surrealdb/surrealql/statements/define/analyzer#tokenizers). 
+Vector Search in SurrealDB is introduced to support efficient and accurate searching of high-dimensional data. 
+This guide will walk you through the essentials of working with vectors in SurrealDB, from storing vectors in embeddings to performing computations and optimizing searches with various indexing strategies.
+
+## What is Vector Search
+
+Vector search is a search mechanism that goes beyond traditional keyword matching and text-based search methods to capture deeper characteristics and similarities between data. 
+
+It converts data such as text, images, or sounds into numerical vectors, called vector embeddings.
+
+You can think of Vector embeddings as cells. Like how cells form the basic structural and biological unit of all known living organisms, vector embeddings serve as the basic units of data representation in vector search. 
+
+Vector search isn't new to the world of data science. [Gerard Salton](https://en.wikipedia.org/wiki/Gerard_Salton), known as the Father of Information Retrieval, introduced the Vector Space Model, cosine similarity, and TF-IDF  for information retrieval around 1960. 
+
+If you’re interested in understanding Vector search in depth, checkout this academic paper on [Vector Retrieval](https://arxiv.org/abs/2401.09350) written by Sebastian Bruch.
+
+## Vector Search vs Full-Text Search
+
+<ThemedImage
+  alt="Google search for the word lead"
+  className="lead screenshot"
+  sources={{
+    light: useBaseUrl('img/lead.png'),
+    dark: useBaseUrl('img/lead.png'),
+  }}
+/>
+This is a screenshot of a Google search for the word “lead”. The search has pulled up different definitions of the word “lead”. 
+Lead can mean `taking initiative`, as well as the chemical element with the symbol `Pb`. 
+
+Now let’s add some context to the word. 
+Consider a database of liquid samples which note down harmful chemicals that are found in them. 
+
+```surql
+INSERT INTO liquids [
+    {sample:'Sea water', content: 'The sea water contains some amount of lead'},
+    {sample:'Tap water', content: 'The team lead by Dr. Rose found out that the tap water in was potable'},
+    {sample:'Sewage water', content: 'High amounts of a were found in Sewage water'}
+]
```

Comment:
What about also providing the DEFINE INDEX and DEFINE ANALYZER statements, so someone can fully try the example?

Reply:
Added define statements for index and analyser.